react-native-libraries.json

```
@@ -9678,5 +9678,13 @@
     "android": true,
     "web": true,
     "expo": true
+  },
+  {
+    "githubUrl": "https://github.com/pcamarajr/react-native-ultimate-icons",
+    "npmPkg": "react-native-ultimate-icons",
+    "examples": ["https://github.com/pcamarajr/react-native-ultimate-icons/tree/main/example/src"],
```

Hello @pcamarajr, thank you for adding new library to the directory! 👍

Overall looks good, just a small nit related to data, before we can merge it in:
```suggestion
    "examples": ["https://github.com/pcamarajr/react-native-ultimate-icons/tree/main/example"],
```

---

Got it. As the npm package name and girhub repository are the same, we can ignore the npmPkg value. Is that it it?

Other that, thank you as well for supporting the `directory`.; 🚀

---

Yup - https://github.com/react-native-community/directory#npmpkg - basically this field is dedicated for scoped packages.